metricsbp/statsd.go

```diff
@@ -313,3 +316,25 @@ func (st *Statsd) Close() error {
 func (st *Statsd) WriteTo(w io.Writer) (n int64, err error) {
 	return st.fallback().statsd.WriteTo(w)
 }
+
+// IncActiveRequests increases the internal active requests counter by 1.
+//
+// The internal active requests counter is used by RunSysStats to report the
+// active_requests gauge.
+//
+// It returns the value after increasing.
+func (st *Statsd) IncActiveRequests() int64 {
```

Comment:
On the statsd object seems like the wrong place to put this, or at least exporting it seems like not what we want to do?

Reply:
statsd object is what reports that gauge, so this sounds like the right place to do it. no?

Reply:
I'm just thinking that I don't think we really want to use this outside of baseplate, adding it onto the Statsd object means it's being put on something that is used throughout services and we don't _really_ want people to use this in their services directly.

Reply:
I can add a comment to say it should only be used by middleware implementations :)